install/updates/40-idp.update

```diff
@@ -2,3 +2,12 @@ dn: cn=idp,$SUFFIX
 default: objectClass: nsContainer
 default: objectClass: top
 default: cn: idp
+
+dn: cn=ipaidpconfiglink,cn=index,cn=userRoot,cn=ldbm database,cn=plugins,cn=config
+only:cn: ipaidpconfiglink
+default:objectClass: nsIndex
+default:objectClass: top
+default:nsSystemIndex: false
+add:nsIndexType: eq
+add:nsIndexType: pres
+add:nsIndexType: sub
```

Comment:
We can also add `referint-membership-attr:ipaidpconfiglink` to install/updates/25-referint.update

Reply:
Good note, added.